deploy/coredns/external-dns.yaml

```diff
@@ -41,6 +41,8 @@ kind: ServiceAccount
 metadata:
   name: external-dns
   namespace: ohmyglb
+imagePullSecrets:
+- name: regcred
```

Comment:
Would a pull secret be a Chart option that defaults to none. I.e. In general use perhaps image pull secrets would not be necessary

Reply:
@donovanmuller that's clearly a bug, thanks for the catch